net/microsocks/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=microsocks
+PKG_VERSION:=1.0.3
+PKG_RELEASE:=1
+PKG_SOURCE:=microsocks-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=http://ftp.barfooze.de/pub/sabotage/tarballs/
+PKG_HASH:=02449160c505ff7ea1bcb71ae781dafcfe66ab95287f3a4fca11621b74b2d7ac
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Mateusz Korniak <matkorgithubcom@ant.gliwice.pl>
+
+include $(INCLUDE_DIR)/package.mk
+
+# Package definition; instructs on how and where our package will appear in the overall configuration menu ('make menuconfig')
+define Package/microsocks
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=SOCKS5 TCP/IP only proxy
+  DEPENDS:=+libc +libpthread
```

Comment:
Are you sure these are needed to be defined?

Reply:
Seems be not needed. Pruned.